dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -169,8 +172,8 @@ def run_ai_evaluations_for_all
 
     script_level = @rubric.lesson.script_levels.find {|sl| sl.levels.include?(@rubric.level)}
 
-    is_ai_experiment_enabled = current_user && Experiment.enabled?(user: current_user, script: script_level.script, experiment_name: 'ai-rubrics')
-    return head :forbidden unless is_ai_experiment_enabled
+    is_teacher_verified = current_user.verified_teacher?
```

Comment:
please use `&.` as in `current_user&.verified_teacher?` throughout, so that we get a 4xx and not a 5xx if the user is not signed in. (ok to skip this if you've already explicitly checked for the presence of the user).